sklearn/mixture/gaussian_mixture.py

```diff
@@ -466,9 +471,10 @@ class GaussianMixture(BaseMixture):
         The user-provided initial means, defaults to None,
         If it None, means are initialized using the `init_params` method.
 
-    covariances_init: array-like, optional.
-        The user-provided initial covariances, defaults to None.
-        If it None, covariances are initialized using the 'init_params' method.
+    precisions_init: array-like, optional.
+        The user-provided initial precisions (inverse of the covariance
+        matrices), defaults to None.
+        If it None, precisions are initialized using the 'init_params' method.
```

Comment:
BTW, I've changed `covariance_init` by `precisions_init`.
You are ok with that or you prefer I keep `covariance_init` ?
I can also propose both possibilities but I don't think it's really clear for the user in that case.